coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -1,47 +1,55 @@
 <template>
-  <div>
-    <ConfirmDialog ref="confirm" />
+  <div id="todo-list">
+    <ConfirmDialog
+      ref="confirm"
+      attach="#todo-list"
+    />
 
     <DeleteErrorDialog
       :dialog="deleteErrorDialog"
       :errors="deleteErrors"
       :warnings="deleteWarnings"
       @okay="resetErrors"
+      attach="#todo-list"
     />
 
     <v-expansion-panels v-if="taskItems && taskItems.length > 0" accordion>
       <v-expansion-panel
-        class="todo-list"
-        v-for="(item, index) in orderBy(taskItems, 'order')"
-        v-bind:key="index"
+        class="align-items-top todo-item"
         expand-icon=""
-        :class="{ 'disabled': !item.enabled, 'draft': isDraft(item),
-        'awaitConfirm': !confirmCheckbox && isConfirmEnabled(item.type, item.status)}">
-
-        <v-expansion-panel-header class="no-dropdown">
+        v-for="(item, index) in orderBy(taskItems, 'order')"
+        :key="index"
+        :class="{
+          'disabled': !item.enabled,
+          'draft': isDraft(item),
+          'awaitConfirm': !confirmCheckbox && isConfirmEnabled(item.type, item.status)
+        }"
+      >
+        <v-expansion-panel-header class="todo-item-toggle no-dropdown">
           <div class="list-item">
-            <div class="filing-type">
+            <div class="todo-label">
               <div class="list-item__title">{{item.title}}</div>
+
               <div class="bcorps-ar-subtitle"
-                 v-if="entityFilter(EntityTypes.BCORP) &&
-                 isConfirmEnabled(item.type, item.status)"
+                v-if="entityFilter(EntityTypes.BCORP) && isConfirmEnabled(item.type, item.status)"
               >
                 <p>Verify your Office Address and Current Directors before filing your Annual Report.</p>
                 <v-checkbox
-                  id="confirm-checkbox"
                   class="todo-list-checkbox"
                   label="All information about the Office Addresses and Current Directors is correct."
                   :disabled=!item.enabled
-                  v-model=confirmCheckbox
+                  v-model="confirmCheckbox"
```

Comment:
Is it always correct to wrap these properties as a string? I know we can pass varying data types, in this case we're passing a boolean, so just curious as to why we are wrapping this?

Reply:
I think there are exceptions but more attribute values should be in quotes. In this case I checked the other places we used v-model and they all had quotes. I changed this and tested it and it seemed to work fine.

Reply:
I definitely agree that based on consistency quotes should be used, I will try and make sure i follow the same pattern as well, Thanks for the comment!